server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -285,3 +285,10 @@ Feature: GetMessages method
     And the preview of the message is "àààà éééé èèèè"
     And the textBody of the message is "àààà\r\n\r\néééé\r\n\r\nèèèè\r\n"
     And the htmlBody of the message is "<html>\r\n  <p>àààà</p>\r\n  <p>éééé</p>\r\n  <p>èèèè</p>\r\n</html>\r\n"
+
+  Scenario: Preview should respect in special cases
+    Given the user has a message "m1" in "INBOX" mailbox with special case
+    When the user ask for messages "m1"
+    Then no error is returned
+    And the list should contain 1 message
+    And the preview of the message is "Afficher l'email dans le navigateur. Hackathon sur les technologies Open Source avec Passerelles Numériques à Danang LINAGORA est fière d'être partenaire de Passerelles Numériques pour animer un Hackathon avec des jeunes étudiants dynamiques et prometteurs"
```

Comment:
I changed to preview should not contain some white spaces characters. I don't know another way to assert this case